tests/qmctl-test/scripts/qmctl_json_output.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# Exit immediately on any error
+set -e
+
+# shellcheck disable=SC1091
+. ../e2e/lib/utils
+
+# Install development tools if needed
+exec_cmd "dnf install --setopt=reposdir=/etc/yum.repos.d  --installroot=/usr/lib/qm/rootfs -y hostname ss || true"
+
+QMCTL_SCRIPT="../../tools/qmctl/qmctl"
+
+info_message "Starting qmctl JSON output tests..."
+
+# Test 1: Show commands with JSON output
+run_test "JSON Show container" 0 "json_valid" "" \
+    python3 "$QMCTL_SCRIPT" show container --json
+
+# Handle unix-domain-sockets separately (may have missing ss command)
+info_message "Testing: JSON Show unix-domain-sockets"
+unix_json_file=$(mktemp)
+python3 "$QMCTL_SCRIPT" show unix-domain-sockets --json > "$unix_json_file" 2>&1
+unix_json_exit=$?
+
+if [ "$unix_json_exit" -eq 0 ]; then
+    if python3 -m json.tool < "$unix_json_file" > /dev/null 2>&1; then
+        pass_message "JSON Show unix-domain-sockets - Valid JSON output"
+    else
```

Comment:
if fail we need to remove unix_json_file

Reply:
Removed